vulnerabilities/models.py

```diff
@@ -144,6 +144,9 @@ class PackageRelatedVulnerability(models.Model):
     vulnerability = models.ForeignKey(Vulnerability, on_delete=models.CASCADE)
     is_vulnerable = models.BooleanField()
 
+    def __str__(self):
+        return self.package.package_url + " " + self.vulnerability.cve_id
```

Comment:
```suggestion
        return f"{self.package.package_url} {self.vulnerability.cve_id}"
```

Also this may not be something we want to have everywhere just for Django admin display.